packages/actions-global/package.json

```diff
@@ -18,8 +18,15 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./index.js",
-  "module": "./index.js",
+  "main": "dist/actions-global.cjs.js",
+  "module": "dist/actions-global.es.js",
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "cross-env npm run build:es && npm run build:cjs",
+    "build:watch": "cross-env npm run build:es -- -w",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./index.js -o dist/actions-global.cjs.js",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./index.js -o dist/actions-global.es.js"
+  },
```

Comment:
We should also provide the `file` option to determine which files gets published. We also do it in the ui-kit (https://github.com/commercetools/ui-kit/blob/8250ebca39ec2a85d8d0d1839013b16cb28b0001/package.json#L18-L29)